pwa/src/components/DiagnosisForm.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { ReactElement } from 'react'
+import { List, Checkbox, InputItem, WhiteSpace } from 'antd-mobile'
+import '../scss/DiagnosisForm.scss'
```

Comment:
This file is currently empty. Is there some content that the file needs or should this import be removed?

Reply:
I'll ask in the stand-up but this could be related to global styling. If so, we should delete this for the time being.

Reply:
My connection dipped out in standup, did we discuss this?

Reply:
We did not, but the file is no longer empty - I added some whitespace styling this morning - so the import is needed.